source/ftrack_api/session.py

```diff
@@ -85,7 +85,7 @@ def __init__(
         self, server_url=None, api_key=None, api_user=None, auto_populate=True,
         plugin_paths=None, cache=None, cache_key_maker=None,
         auto_connect_event_hub=False, schema_cache_path=None,
-        plugin_arguments=None, timeout=60
+        plugin_arguments=None, timeout=60, cookies=None, headers=None, ftrack_strict_api=False
```

Comment:
maybe we can call the keyword argument just "strict_api"

Reply:
Right, no problem.